sound/soc/sof/intel/hda.c

```diff
@@ -47,6 +47,12 @@ struct hda_dsp_msg_code {
 	const char *msg;
 };
 
+static bool hda_use_msi = IS_ENABLED(CONFIG_PCI);
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG)
+module_param_named(use_msi, hda_use_msi, bool, 0444);
+MODULE_PARM_DESC(use_msi, "SOF PCI use MSI mode");
```

Comment:
SOF HDA use PCI MSI mode? This isn't a generic PCI thing.

Reply:
@plbossart updated